stolostron/Dockerfile.stolostron

```
@@ -19,11 +19,14 @@ RUN  --mount=type=cache,target=/root/.local/share/golang \
      --mount=type=cache,target=/go/pkg/mod \
      go mod download
 
+# Install static glibc libs required for CGO_ENABLED=1 static linking
+RUN yum install -y glibc-static && yum clean all
```

Hadolint (2.14.0)

[warning] Specify version with `yum install -y <package>-<version>`.

(DL3033)